data-reporting-service/observation-service/src/main/java/gov/cdc/etldatapipeline/observation/controller/ObservationController.java

```diff
@@ -0,0 +1,23 @@
+package gov.cdc.etldatapipeline.observation.controller;
+
+import gov.cdc.etldatapipeline.observation.service.KafkaProducerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ObservationController {
+    private final KafkaProducerService producerService;
+    private final String topicName = "cdc.nbs_odse.dbo.Observation";
```

Comment:
@rshanmugam-enquizit Move topicName to its own constant file in java

Reply:
this will be moved as a part of common util module